docs/src/content/docs/getting-started/breaking-changes.mdx

```diff
@@ -7,6 +7,68 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 Features get added and removed, and breaking changes are introduced! This documents how to migrate.
 
+## `0.10.0`
+
+---
+
+### A better way for development HMR using `hmrIntegration`
+
+Until now, we recommended `watchIntegration` to enable HMR in development. However:
+- This can cause perf issues because paths would be also watched once the integration is consumed from npm
+- It doesn't work with a build step, that we highly recommend.
+
+From now on, we recommend that you use dynamic imports in your local playgrounds, alongside the new `hmrIntegration`
+exported from `astro-integration-kit/dev`.
+
+```ts title="package/src/integration.ts" del={3-4,10,13}
+import {
+    defineIntegration,
+    createResolver,
+    watchIntegration
+} from "astro-integration-kit"
+
+export const integration = defineIntegration({
+    // ...
+    setup() {
+        const { resolve } = createResolver(import.meta.url)
+        return {
+            "astro:config:setup": (params) => {
+                watchIntegration(params, resolve())
+            }
+        }
+    }
+})
+```
+
+```ts title="playground/astro.config.mjs" del={2} ins={3-5,10-12}
+import { defineConfig } from "astro/config";
+import packageName from "package-name"
+import { createResolver } from "astro-integration-kit";
+import { hmrIntegration } from "astro-integration-kit/dev";
+const { default: packageName } = await import("package-name");
+
+export default defineConfig({
+	integrations: [
+		packageName(),
+        hmrIntegration({
+            directory: createResolver(import.meta.url).resolve("../package/dist")
+        })
+	],
+});
+```
```

Comment:
This feels a bit out of place, it is duplicated from `/hmrIntegration` and it is more of an addition/improvement instead of a breaking change to existing code like all the other code snippets. Maybe it would be better to add a note or something in the section below, redirecting people to `hmrIntegration` when using `watchDirectory` for package development. Another option could be to just rename the page to "Upgrade Guide" instead of "Breaking Changes".

Reply:
I initially wrote this and then copy pasted in `/dev/hmr-integration/`. Yeah you're suggestion makes sense, let me do that!